sync-core/src/test/java/com/cloudant/sync/datastore/DatastoreManagerTest.java

```diff
@@ -142,28 +151,38 @@ public void createDatastoreWithForwardSlashChar() throws Exception {
 
     @Test
     public void list5Datastores() throws Exception {
+        List<Datastore> opened = new ArrayList<Datastore>();
 
-        for (int i = 0; i < 5; i++) {
-            manager.openDatastore("datastore" + i);
-        }
+        try {
```

Comment:
Could this be done more concisely (without the `opened` list) by putting the try inside the loop?